pkg/sql/colexec/expr_eval.go

```diff
@@ -190,11 +190,13 @@ func getConstVec(ctx context.Context, proc *process.Process, expr *plan.Expr, le
 		case *plan.Const_Decimal64Val:
 			cd64 := t.C.GetDecimal64Val()
 			d64 := types.Decimal64FromInt64Raw(cd64.A)
-			vec = vector.NewConstFixed(constDecimal64Type, length, d64, proc.Mp())
+			typ := types.Type{Oid: types.T_decimal64, Size: 8, Width: expr.Typ.Width, Scale: expr.Typ.Scale}
```

Comment:
Should use `types.New(types.T_decimal64, width, scale)` here

Reply:
done